### DISCUSSION TITLE
github css misplacement

### DISCUSSION THREAD

**Discussion by sbfkcel** [community/community#29769] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/5469785/185066348-455d69b8-7a49-4b81-ab2f-1ac08572c328.png)

**Comment by melvincwng** [comment]
@sbfkcel , You can report bugs to GitHub over [here](https://support.github.com/contact/bug-report).

Hope this helps

**Comment by dylanatsmith** [comment, marked as answer]
Thanks for the report. We’re aware of the issue and should be shipping a fix shortly.

**Reply by ritik619** [reply]
Hey @dylanatsmith It's just a tiny issue. We need to adjust the z-index.

**Reply by ritik619** [reply]
Would be happy to fix it.
